uaclient/entitlements/esm.py

```diff
@@ -16,31 +11,4 @@ class ESMEntitlement(repo.RepoEntitlement):
         ' (https://ubuntu.com/esm)')
     repo_url = 'https://esm.ubuntu.com'
     repo_key_file = 'ubuntu-esm-v2-keyring.gpg'
-
-    def disable(self, silent=False, force=False):
```

Comment:
Moved ESMEntitlement.disable and FIPSEntitlement.disable to common parent RepoEntitlement.disable.